spec/betterc.dd

```diff
@@ -0,0 +1,48 @@
+Ddoc
+
+$(SPEC_S Better C,
+
+`-betterC` is a Command-Line flag for `dmd`, which generated light-weight,
+barebone, runtime-free binaries with are suitable for linking from C.
+
+---
+void main()
+{
+    import core.stdc.stdio : printf;
+    printf("Hello betterC");
+}
+---
+
+$(CONSOLE
+> dmd -betterC hello.d
+Hello betterC
+)
+
+$(H2 $(LNAME2 effects, Effects))
+
+    $(OL
+        $(LI Doesn't link DRuntime and standard library by default)
+	    $(LI Doesn't generate ModuleInfo entries)
+    )
+
+$(H2 $(LNAME2 consequences, Consequences))
+
+As no DRuntime is available, many D features won't work. For example:
+
+	$(OL
+	    $(LI No static module constructors or deconstructors)
+	    $(LI No Garbage Collection)
+	    $(LI No dynamic arrays and associative arrays)
+	    $(LI No Classes)
+	    $(LI No `synchronized`, monitors, mutex from $(MREF core, sync))
+	    $(LI No `shared` (`__gshared` can be used instead))
+	    $(LI No $(MREF core, thread))
+	    $(LI No built-in instrinsics (e.g. $(MREF core, math)))
```

Comment:
I actually tried this quickly with `-betterC`:

```d
void main() {
    import core.stdc.stdio : printf;
    import core.math : cos;
    printf("Hello world: %.2f", cos(2));
}
```

and got a segfault. However, I simply forgot `extern(C)` for the main method.